app/models/katello/concerns/operatingsystem_extensions.rb

```diff
@@ -3,15 +3,25 @@ module Concerns
     module OperatingsystemExtensions
       extend ActiveSupport::Concern
 
+      REDHAT_ATOMIC_HOST_DISTRO_NAME = "Red Hat Enterprise Linux Atomic Host"
+      REDHAT_ATOMIC_HOST_OS = "RedHat_Enterprise_Linux_Atomic_Host"
```

Comment:
Would we want to say "Red_Hat" versus "RedHat" here or is this a value that we need to match on from an external input?